Vagrantfile

```diff
@@ -129,6 +134,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     end
 
     app.vm.provision "ansible" do |ansible|
+      ansible.extra_vars = extra_vars
```

Comment:
Will that work here? I thought `lookup` gets environment variables on the guest machine, not the host machine.